HelpSource/Classes/CombC.schelp

```diff
@@ -63,6 +63,17 @@ code::
 // used as an echo.
 { CombN.ar(Decay.ar(Dust.ar(1,0.5), 0.2, WhiteNoise.ar), 0.2, 0.2, 3) }.play;
 
+({
+ 	var in, del1, del2, del3, del4;
```

Comment:
Also, isn't this the example from below the "// large delay lines" comment? If so we shouldn't put this code before that note